hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
         """HiPS access url"""
-        return self.data['moc_access_url'].rsplit('/', 1)[0]
+        try:
+            return self.data['hips_service_url']
+        except IndexError:
```

Comment:
are you sure you ever enter in this except branch?
If the key `hips_service_url` is missing, isn't a `KeyError` raised rather than a `IndexError`?